### PR TITLE
Full website build out for Cincy

### PR DESCRIPTION
**Pull request by ericholscher**
This creates a full website build for the Cincy conf. There is a lot we need to update here, but it at least gets us started. 

I think we can trim down almost all the content pages and kill the NA/EU specific stuff. We should be backporting most of it into YAML, but for now we can just ship it with basic info. 

Needs:

* Venue info
* Open Help info
* Schedule YAML updates
* Logo cobrand updates
* Hiding of a subset of pages for now
* Index page cleanup
* Header image changes

### CODE REVIEW

**Comment by ericholscher** [comment]
Preview: https://deploy-preview-493--dentist-canary-64055.netlify.com/conf/cincinnati/2018/

File: docs/_templates/2018/index.html [hunk 1]
```diff
@@ -91,11 +91,22 @@
...
         <h2>What is Write the Docs about?</h2>
         <p>Write the Docs brings everyone who writes the docs together in the same room: Programmers, Tech Writers, Support, Designers, Developer Advocates, and more. We all have things to learn from each other, and there’s no better way than sitting together and talking.</p>
         <p>We invite you to join {{ about.attendees }} other folks in our event to explore the art and science of documentation.</p>
+
+        {% if cobrand.logo %}
+        <h2>What is Open Help about?</h2>
+        <p>The Open Help Conference features two days of presentations and open discussions. In true open source spirit, our discussions and agenda are led by our attendees, allowing you to learn and share what matters to you most. With some of the most interesting people in the industry, the Open Help Conference showcases the ideas that are shaping the future of help.</p>
+
+        <h2>What is this event?</h2>
+        <p>This event is a combination of the two communities and conferences. We will have a one day Write the Docs style talk track, followed by a combination day of discussions. Then we'll do an Open Help style 3-day sprint.</p>
```
Comment (plaindocs) [inline docs/_templates/2018/index.html]: "combination day" ??

File: docs/_templates/2018/index.html [hunk 2]
```diff
@@ -91,11 +91,22 @@
...
   <div class="about">
     <div class="uk-container">
+
       <div class="about-content">
         <h2>What is Write the Docs about?</h2>
         <p>Write the Docs brings everyone who writes the docs together in the same room: Programmers, Tech Writers, Support, Designers, Developer Advocates, and more. We all have things to learn from each other, and there’s no better way than sitting together and talking.</p>
         <p>We invite you to join {{ about.attendees }} other folks in our event to explore the art and science of documentation.</p>
+
+        {% if cobrand.logo %}
+        <h2>What is Open Help about?</h2>
```
Comment (plaindocs) [inline docs/_templates/2018/index.html]: would these headings, inc WTD, be better without "about"?

File: docs/conf/cincinnati/2018/about.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
+:template: {{year}}/generic.html
+:banner: _static/{{year}}/assets/headers/team.png
+
+About the Conference
+====================
+
+We invite you to join hundreds of other folks for a three-day event to explore the art and science of documentation.
```
Comment (plaindocs) [inline docs/conf/cincinnati/2018/about.rst]: It ain't a three day event

Comment (plaindocs) [reply docs/conf/cincinnati/2018/about.rst]: We should add OH stuff here also?

File: docs/conf/cincinnati/2018/cfp.rst [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+----------------
+
+Write the Docs conferences are our chance to get together in-person to explore and celebrate the craft of documentation in a positive, inclusive environment.
+
+Between the talks on stage, the discussions in the unconference, the collaboration on the writing day and the conversations in the hallway, 100% of the content for our conferences comes from our community! Whether your job title is writer, developer, product manager, support advocate, librarian, or one of a hundred others, your perspective on what makes good docs is what this conference is all about.
+
+Our goal is to give documentarians a chance to connect and learn from each other. You'll have the chance to compare notes on what's happening in the industry, dig into questions of convention and good practice, and generally nerd out about all the things we love about documentation.
+
+What we’re looking for
+----------------------
```
Comment (plaindocs) [inline docs/conf/cincinnati/2018/cfp.rst:29]: Are we looking for the same as NA and EU? I guess we are, just less of it on the schedule?

File: docs/conf/cincinnati/2018/lightning-talks.rst [hunk 5]
```diff
@@ -0,0 +1,9 @@
+:template: {{year}}/generic.html
+:banner: _static/2018/assets/headers/venue.png
+
+Lightning Talks
+===============
+
+.. _speaker-lightning-talks-2018/eu:
```
Comment (plaindocs) [inline docs/conf/cincinnati/2018/lightning-talks.rst]: ooh, sneaky nice!

File: docs/conf/cincinnati/2018/news/prague-18-welcome.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
+:template: {{year}}/generic.html
+
+.. post:: Mar 15, 2018
+   :tags: prague-2018, website, tickets, cfp
+
+Announcing Prague Website, CFP, and Ticket Sales
```
Comment (plaindocs) [inline docs/conf/cincinnati/2018/news/prague-18-welcome.rst]: We just need to clobber this with #485

File: docs/conf/cincinnati/2018/welcome-wagon.rst [hunk 7]
```diff
@@ -0,0 +1,260 @@
+:template: {{year}}/generic.html
+:banner: _static/2018/assets/headers/group.png
+
+Welcome Wagon
+=============
```
Comment (plaindocs) [inline docs/conf/cincinnati/2018/welcome-wagon.rst]: We're gonna need more writing sprint specifc WW stuff for three days

**Review by plaindocs — COMMENTED** [review]
Great start!

Let me know if you want me to help on any of the remaining converting

The CTA buttons need some very minor CSS tweaking, they are up against the black spearator line due to the OH stuff.

File: docs/_templates/2018/index.html (hunk 1 above)
Comment (hillaryfraley) [reply docs/_templates/2018/index.html]: Or maybe "This event combines the two communities and conferences."

File: docs/_templates/2018/index.html (hunk 2 above)
Comment (hillaryfraley) [reply docs/_templates/2018/index.html]: I agree -- "What is Write the Docs?" and "What is Open Help?" sound better to me.

File: docs/_templates/2018/index.html (hunk 1 above)
Comment (hillaryfraley) [reply docs/_templates/2018/index.html]: Maybe revise the other two sentences to:

We will have one day of Write the Docs-style talks, followed by one day of combined Write the Docs and Open Help discussions. Then we'll have an Open Help-style three-day documentation sprint.

**Pull request merged**